crypto/stack/stack.c

```diff
@@ -250,7 +250,7 @@ void OPENSSL_sk_pop_free(OPENSSL_STACK *st, OPENSSL_sk_freefunc func)
         return;
     for (i = 0; i < st->num; i++)
         if (st->data[i] != NULL)
-            func(st->data[i]);
+            func((char *)st->data[i]);
```

Comment:
Suppose it's something with embedded pointers, like an X509.  I believe the code is correct.